django_mongodb/features.py

```diff
@@ -82,19 +93,26 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         "ordering.tests.OrderingTests.test_order_by_parent_fk_with_expression_in_default_ordering",
         "ordering.tests.OrderingTests.test_order_by_ptr_field_with_default_ordering_by_expression",
         "queries.tests.Queries1Tests.test_order_by_tables",
-        "queries.tests.Queries1Tests.test_ticket4358",
         "queries.tests.TestTicket24605.test_ticket_24605",
         "queries.tests.TestInvalidValuesRelation.test_invalid_values",
         # alias().order_by() doesn't work.
         "annotations.tests.AliasTests.test_order_by_alias",
         "annotations.tests.AliasTests.test_order_by_alias_aggregate",
         # annotate() + values_list() + order_by() loses annotated value.
         "expressions_case.tests.CaseExpressionTests.test_annotate_values_not_in_order_by",
-        # pymongo.errors.OperationFailure: the limit must be positive
-        "queries.tests.WeirdQuerysetSlicingTests.test_tickets_7698_10202",
         # QuerySet.explain() not implemented:
         # https://github.com/mongodb-labs/django-mongodb/issues/28
         "queries.test_explain.ExplainUnsupportedTests.test_message",
+        # The $sum aggregation returns 0 instead of None for null.
+        "aggregation.test_filter_argument.FilteredAggregateTests.test_plain_annotate",
+        "aggregation.tests.AggregateTestCase.test_aggregation_default_passed_another_aggregate",
+        "aggregation.tests.AggregateTestCase.test_annotation_expressions",
+        "aggregation.tests.AggregateTestCase.test_reverse_fkey_annotate",
+        # Manage empty result when the flag elide_empty is False
```

Comment:
Is this a TODO even though the test probably won't work with SQL-specific: `Func("book", function="COUNT")`?

Reply:
Yes, it is a TODO, but maybe this test is not the right to do that. I think, this should be moved to the raw sql queries...